Overview.src.html

```diff
@@ -3944,11 +4021,22 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <span title=concept-request-body>body</span> is non-null, run these substeps:
 
   <ol>
+   <li><p>Let <var>stream</var> be an <span title=concept-empty-ReadableStream>empty</span>
+   <code title=concept-ReadableStream>ReadableStream</code> object.
+
    <li><p>Set <var>input</var>'s <span title=concept-Request-request>request</span>'s
-   <span title=concept-request-body>body</span> to an empty byte stream.
+   <span title=concept-request-body>body</span> to a new <span title=concept-body>body</span> whose
+   <span title=concept-body-stream>stream</span> is <var>stream</var>.
 
-   <li><p>Set <var>input</var>'s
-   <span title=concept-Request-disturbed-flag>disturbed flag</span>.
+   <li>
```

Comment:
```
2. Let temporaryBody be input's request's body if input is a Request object, and null otherwise.
...
33. Set r's request's body to temporaryBody.
```